examples/src/App.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Cognite AS
+ */
+
+import React from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import { routes } from "./routes";
+import { Container } from "./components/styled";
+
+export default function App() {
+  return (
+    <Router>
+      <div
+        style={{
+          padding: "5px",
+        }}
+      >
+        <div
+          style={{
+            boxShadow: "rgba(0, 0, 0, 0.1) 0px 1px 5px 0px",
+            background: "rgb(255, 255, 255)",
+            padding: "15px",
+            borderRadius: "4px",
+            minHeight: "95vh",
+            display: "flex",
+            flexDirection: "column",
+            overflowX: "auto",
+          }}
+        >
+          <Switch>
```

Comment:
Should we define the cognite client outside of the examples since I am guessing we are using react reroute and don't actually reload pages on url change?

Reply:
Yep, maybe later. We can declare it once here and then pass with context or prop.